### PR TITLE
Sidebar and Footer component

### PR DESCRIPTION
**Pull request by jorgemasta**
- Create a new WizardSidebar component
- Modify other wizard-related components to adjust to new styles

Screenshot 2022-11-25 at 17 52 45

### CODE REVIEW

**Review by chumpgrub — APPROVED** [review]
I wasn't sure if you were ready for a code review on this, but I took a look anyways! The WizardSidebar is fantastic and the WizardFooter is looking nice, too. I **do** have a couple questions on the WizardFooter after poking at that component for a few minutes.

File: packages/wme-ui/src/components/wizard-footer/wizard-footer.tsx [hunk 1]
```diff
@@ -237,14 +245,14 @@ const WizardFooter: React.FC<WizardFooterProps> = (props) => {
               }
               {
                 isLoading ? (
-                  <LoadingButton
+                  <StyledLoadingButton
                     loading
                     variant="contained"
                     loadingPosition="start"
                     startIcon={<ChevronRight />}
```
[the remark this answers is not in the document]
Comment (jorgemasta) [reply packages/wme-ui/src/components/wizard-footer/wizard-footer.tsx]: Cool, that makes sense!
Refactored.

File: packages/wme-ui/src/components/wizard-footer/wizard-footer.tsx [hunk 2]
```diff
@@ -237,14 +245,14 @@ const WizardFooter: React.FC<WizardFooterProps> = (props) => {
...
                   >
                     {loadingText}
-                  </LoadingButton>
+                  </StyledLoadingButton>
                 )
                   : (
                     <Button
```
[the remark this answers is not in the document]
Comment (jorgemasta) [reply packages/wme-ui/src/components/wizard-footer/wizard-footer.tsx:263]: I have added 2 new props to the `Footer` component:
- `nextEndIcon`
- `nextSrtartIcon`

With these props, we can add the icons we need.

[the remark this answers is not in the document]
Comment (chumpgrub) [reply packages/wme-ui/src/components/wizard-footer/wizard-footer.tsx:263]: Love it! Thank you!

**Pull request merged**